### DISCUSSION TITLE
EJBCA + Mysql = No initialization of SuperAdmin account

### DISCUSSION THREAD

**Discussion by lunitaris** [Keyfactor/ejbca-ce#361] [General]
Hi, 

I am trying to get EJBCA to work with containers.
I am using podman, witch is kind of the same of docker in terms of cmdlines.

I built 3 containers:
- Nginx
- Mysql Db
- EJBCA.

I am launching EJBCA, it connects to my external MySQL database correctly, creates all the tables needed.
The thing is:

Comparing to laucnhing EJBCA without an external Mysql DB, EJBCA automatically creates the Management CA and the superadmin role.
But when I use an external DB, the initialization of Management CA / SuperAdmin is not done.
So  when I go to the front page, I cannot login / enroll the SuperAdmin User.

On EJBCA RA, I have this message:
_The service is currently unavailable since no active CA is serving this RA._

What can I do, to make EJBCA initialize the first ManagementCA, and SuperAdmin account?

**Comment by primetomas** [comment]
How the container starts up, if it creates an initial ManagementCA or not is controlled by the environment variable TLS_SETUP_ENABLED. See https://hub.docker.com/r/keyfactor/ejbca-ce
(unless you created your own container in which case it's all under your control)

**Comment by lunitaris** [comment]
Yes indeed.

I found that it is related to PROXY_HTTP_BIND variable.
When I set it to 127.0.0.1, (because I have a reverse proxy: Nginx), there is no generation of SuperAdmin Account.
I removed this variable and made my reverse proxy connect to classic ports (8080/8443).
